portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACNavigation.macro

```diff
@@ -60,4 +60,12 @@ definition {
 		Click(locator1 = "ACNavigation#TAB_NAME");
 	}
 
+	macro changeTimeSelector {
```

Comment:
Hey @ViniLopes20, I believe we have a similar macro in place in [ACTimeFilter.macro](https://github.com/alexcandidos/liferay-portal/blob/master/portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACTimeFilter.macro)